optimum/intel/openvino/modeling.py

```diff
@@ -473,6 +473,35 @@ def _prepare_attention_mask_for_generation(
             # Ensure attention mask is on the same device as the input IDs
             return torch.ones(inputs.shape[:2], dtype=torch.long, device=inputs.device)
 
+    def _reshape(
+        self,
+        model: openvino.runtime.Model,
+        batch_size: int,
+        sequence_length: int,
+        height: int = None,
+        width: int = None,
+    ):
+        if batch_size != -1:
+            logger.warning(
+                f"The batch size was set to {batch_size} during reshape which is not supported for causal language model."
```

Comment:
Since both dynamic batch and dynamic sequence length are not supported, I'm wondering if it would be clearer to show one message when a user tries to reshape the model to static shapes, that causal language models need dynamic shapes and that reshaping is not supported. What do you think?

Reply:
I updated the warning message